src/components/InfoTooltip/InfoTooltip.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import * as Tooltip from '@radix-ui/react-tooltip';
+import { Info } from '@phosphor-icons/react';
+import './InfoTooltip.css';
+
+interface InfoTooltipProps {
+  content: string;
+}
+export const InfoTooltip = (props: InfoTooltipProps) => {
+  return (
+    <Tooltip.Provider>
+      <Tooltip.Root>
+        <Tooltip.Trigger asChild>
+          <Info className='info-tooltip-icon' />
+        </Tooltip.Trigger>
+        <Tooltip.Portal>
+          <Tooltip.Content className='info-tooltip-content' sideOffset={5}>
```

Comment:
Re above: I think if you replace `info-tooltip-content` with `tooltip-content` and `info-tooltip-arrow` with `tooltip-arrow`, our base styles should kick in.

Reply:
I will make that change.